Modules/_interpretersmodule.c

```diff
@@ -1533,18 +1504,21 @@ interp_decref(PyObject *self, PyObject *args, PyObject *kwds)
 }
 
 
+/*[clinic input]
+_interpreters.capture_exception
+    exc_arg: object(c_default='NULL') = None
+    /
+
+Return a snapshot of an exception.
+
+If *exc* is None then the current exception, if any, is used (but not cleared).
```

Comment:
This is not Markdown or reStructuredText. `*` does not work as a mark up.

Reply:
The current docstring uses `"exc"`. There are also other cases above.

Reply:
Now using `"exc"` etc